src/utils.ts

```diff
@@ -8,7 +8,7 @@ export function isPayloadMethod(method = "GET") {
 }
 
 export function isJSONSerializable(value: any) {
-  if (value === undefined) {
+  if (value === undefined || value.buffer) {
```

Comment:
This is unsafe before checking other falsy cases we do in L15 (false and null). If value is `null`, `value.buffer` makes an error

Reply:
I see. Thanks @pi0